### PR TITLE
Fixed illegal character problem

### PR DESCRIPTION
**Pull request by SebastianIoanPop**

### CODE REVIEW

**Comment by ElvenSpellmaker** [comment]
:shipit:

**Comment by ElvenSpellmaker** [comment]
@SebastianIoanPop Shouldn't there be tests for this functionality btw?

File: index.js [hunk 1]
```diff
@@ -2,6 +2,9 @@
 let defaultInterval = 10;
 let oneSecInMs = 1000;
 let keyCountAppendName = 'keyCounter';
+let statsdIllegalCharacter = ':';
+let defaultWordSeparatorCharacter = '-';
```
Comment (nicolaspujol) [inline index.js]: You might want to use a const for that

File: index.js [hunk 2]
```diff
@@ -50,7 +53,9 @@ redisClients.forEach((c) => {
         return;
       }
 
-      statsdClient.gauge(`${prefix}${stat.key}${suffix}`, stat.count);
+      var cleanKey = stat.key.replace(statsdIllegalCharacter, defaultWordSeparatorCharacter);
```
Comment (nicolaspujol) [inline index.js]: var declaration should be at the top of the code block + use `let`

**Review by magicprog-web — CHANGES_REQUESTED** [review]
Added comments

File: index.js [hunk 3]
```diff
@@ -44,13 +45,14 @@ redisClients.forEach((c) => {
...
       var suffix = libUtils.getSuffix(c, keyCountAppendName);
+      let cleanKey = libUtils.cleanKey(stat.key);
 
       if (err) {
         util.log(`[${c.host}] ${err}`);
         return;
       }
-
-      statsdClient.gauge(`${prefix}${stat.key}${suffix}`, stat.count);
+
```
Comment (nicolaspujol) [inline index.js]: remove spaces/tab

**Review by magicprog-web — APPROVED** [review]

**Pull request merged**